client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -194,23 +179,20 @@ const Security2faSMSSettings = createReactClass( {
 					<div className="security-2fa-sms-settings__fieldset-container">
 						<QuerySmsCountries />
 						<FormPhoneInput
-							ref="phoneInput"
 							countriesList={ this.props.countriesList }
-							disabled={ this.state.submittingForm }
+							disabled={ this.props.isUpdatingUserSettings }
 							countrySelectProps={ {
-								onFocus: function () {
+								onFocus() {
 									gaRecordEvent( 'Me', 'Focused On 2fa SMS Country Select' );
 								},
 							} }
 							phoneInputProps={ {
-								onFocus: function () {
+								onFocus() {
 									gaRecordEvent( 'Me', 'Focused On 2fa SMS Phone Number' );
 								},
 							} }
-							initialCountryCode={ this.props.userSettings.getSetting( 'two_step_sms_country' ) }
-							initialPhoneNumber={ this.props.userSettings.getSetting(
-								'two_step_sms_phone_number'
-							) }
+							initialCountryCode={ this.props.userSettings.two_step_sms_country }
```

Comment:
These can now be `undefined`, while before they were `null` if the user settings aren't fetched yet and `getSetting()` was used. Any concerns with that change?

Reply:
I don't think this should be an issue because this component is only shown as the second step. At that point `userSettings` are already there because otherwise the user couldn't click on _Get started_ (due to the changes in #49002).

Reply:
Thanks for confirming 👍